src/routes/lend.js

```diff
@@ -187,9 +203,10 @@ function LendPage() {
     const closeListingModal = useCallback((didListNFT) => {
         setListingModalState({ isShown: false, tokenID: '', tokenAddress: '' });
         if (didListNFT) {
+            fetchAvailableListings();
```

Comment:
And you probably don't even need to parse the event (just need to know the tx has been confirmed) - since you already know the tokenID and tokenAddress here, before clearing the Modal state.